projects/batfish/src/org/batfish/grammar/cisco/CiscoParser.g4

```diff
@@ -1,6 +1,6 @@
 parser grammar CiscoParser;
 
-import
```

Comment:
okay. Highlight whole file, right click, source, format? or is there an easier way to do it?